test.py

```diff
@@ -0,0 +1,10 @@
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+
+
+class A:
+    def b():
+        my_name_is_model = dpf.Model(examples.msup_transient)
+        my_name_is_model.results.displacement.on_last_time_freq.on_named_selection(
+            "_CONSTRAINEDNODES"
+        )
```

Comment:
Curious, why was this added?

Reply:
This was a test module I made to properly format a docsring example that was too long. It should not have been committed. I will remove it!